external-crates/move/crates/move-compiler/tests/move_2024/parser/positional_struct_explicit_type_arg_assign.exp

```diff
@@ -1,3 +1,38 @@
+error[E03003]: unbound module member
```

Comment:
The changes to this file make me nervous. Can you explain why this occurred? Can we fix it?